list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries with user input, which is vulnerable to SQL injection attacks.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements to prevent SQL injection. For example, use `mysqli_prepare` and `mysqli_stmt_bind_param`.
- **Example**:
  ```php
  $stmt = $connection->prepare("SELECT * FROM books");
  $stmt->execute();
  $result = $stmt->get_result();
  ```